deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node-create.sh

```diff
@@ -5,4 +5,4 @@ ZK_NAMESPACE="${ZK_NAMESPACE:-zoo1ns}"
 CUR_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
 kubectl create namespace "${ZK_NAMESPACE}"
-kubectl --namespace="${ZK_NAMESPACE}" apply -f "${CUR_DIR}/zookeeper-1-node.yaml"
+cat "${CUR_DIR}/zookeeper-1-node.yaml" | kubectl --namespace="${ZK_NAMESPACE}" apply -f -
```

Comment:
This change is not needed. kubectl apply -f <filename.yaml> is a kubernetes way.